packages/strapi-parts/src/Tabs/Tabs.js

```diff
@@ -1,11 +1,18 @@
 import React, { Children, cloneElement } from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import { useTabs } from './TabsContext';
 import { ButtonText, TableLabel } from '../Text';
 import { KeyboardKeys } from '../helpers/keyboardKeys';
 import { useTabsFocus } from './useTabsFocus';
 import { DefaultTabsRow, DefaultTabButton, DefaultTabBox, SimpleTabBox } from './components';
 
+const TabButton = styled.button`
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+  }
+`;
```

Comment:
Why do we need this modification ?

Reply:
If we do this here, don't we need to standardize it in every disabled components ? @maevalienard